src/Common/src/System/NonSerializedAttribute.cs

```diff
@@ -5,10 +5,10 @@
 namespace System
 {
     [AttributeUsage(AttributeTargets.Field, Inherited = false)]
-    public sealed class NonSerializedAttribute : Attribute
```

Comment:
No, this source isn't used anywhere today. The actual public implementation is in coreclr and corert.